### PR TITLE
Feature/style variables

### LINKED ISSUE #11: [style] Create variables for styling
Implement variables for:

- colors;
- fonts;
- font-sizes.

Implement mixins for media queries (see *SRS (design):

- desktop view;
- mobile view.
### Estimation: 1h

variables

SRS - [SRS](https://docs.google.com/document/d/1Z0ceG_NtqwGXs6xRjqq8hb9AoqWj1lLzboL5epCSfkQ/edit#)

### PR DESCRIPTION
**Pull request by solomiia-sahala**
Created style variables with colors, fonts, font sizes and mixin for mobile and desktop.

### CODE REVIEW

File: src/styles/variables.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+$dark-grey: #17252A;
+
+//fonts
+$title: "Fira Sans", sans-serif;
+$main-font: "Source Sans Pro", sans-serif;
+
+//font sizes
+$font-size-14-mobile: 14px;
+$font-size-18-mobile: 18px;
+$font-size-24-title-mobile: 24px;
```
Comment (alexandrbig) [inline src/styles/variables.scss]: what if the title size is changed to 26px for example, will you change the variable name also?

Comment (solomiia-sahala) [reply src/styles/variables.scss]: i renamed the variables for font sizes, so they are more general and not tied to size (s - small, m - medium, l - large)

Comment (solomiia-sahala) [reply src/styles/variables.scss]: cause changing the variable name will case to change code in each place we use it.

File: src/styles/variables.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//colors
+$mint: #3AAFA9;
+$dark-green: #2B7A78;
+$light-green: #DEF2F1;
+$bg-color-light: #FEFFFF;
+$dark-grey: #17252A;
+
+//fonts
+$title: "Fira Sans", sans-serif;
+$main-font: "Source Sans Pro", sans-serif;
```
Comment (alexandrbig) [inline src/styles/variables.scss:10]: why $title and $main-font have similar value but different naming?
could you stick to the same naming please?

**Review by alexandrbig — APPROVED** [review]

**Pull request merged**